### ISSUE TITLE
Autocomplete search input

### ISSUE THREAD

**Issue by jhta** [anshumanv/evolutions-webapp#48]
##### This issue is about {Autocomplete search input}

- [ x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x ] Assign this issue to me (I want to work on this).
- [ x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [ x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

I would like add some autocomplete search input to this project, what do you think @anshumanv ?

**Comment by anshumanv** [comment]
This would be amazing, please go ahead! :pray: :sparkles:

**Comment by jhta** [comment]
sure, on it :)
